go-app-ussd_optout.js

```diff
@@ -130,29 +130,29 @@ go.app = function() {
                             "to help you in this difficult time?"),
 
                 choices: [
-                    new Choice("state_send_loss_optout", $("Yes")),
-                    new Choice("state_send_nonloss_optout", $("No"))
+                    new Choice("state_send_loss_switch", $("Yes")),
+                    new Choice("state_send_loss_optout", $("No"))
                 ],
 
                 next: function(choice) {
-                    if (choice.value == "state_send_loss_optout") {
-                        return hub
-                        .create_change(
-                            {
-                                "registrant_id": self.im.user.answers.operator.id,
-                                "action": "momconnect_loss_switch",
-                                "data": {
-                                    "reason": self.im.user.answers.state_start
-                                }
-                            }
-                        )
-                        .then(function() {
-                            return choice.value;
-                        });
-                    } else {
-                        return choice.value;
+                    return choice.value;
```

Comment:
Since we are removing all this logic in the `next` function, we can use a `MenuState` instead of a `ChoiceState` and avoid the `next` function altogether.